packages/create-fuels/src/cli.ts

```diff
@@ -66,7 +60,11 @@ export const runScaffoldCli = async ({
   program.parse(args);
 
   let projectPath = program.args[0] ?? (await promptForProjectPath());
-  const verboseEnabled = program.opts().verbose ?? false;
+
+  const opts = program.opts<ProgramOptions>();
+  const verboseEnabled = opts.verbose ?? false;
+  const packageManager = await getPackageManager(opts);
+  const packageManagerCommand = packageMangerCommands[packageManager];
```

Comment:
IMO this should be part of the `packageManager` object so that below it's used like this:
```ts
execSync(packageManager.install, { stdio: verboseEnabled ? 'inherit' : 'pipe' });

log(`- Start a local Fuel dev server: ${packageManager.run('fuels:dev')}`);

log(`- Run the frontend: ${packageManager.run('dev')}`);
```

I can live with the current implementation, but this is cleaner.